website/docs/sdks/index.md

```diff
@@ -49,55 +49,55 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
-| Can block until synchronized                                                                      | ✅                     | ✅                        | ⭕                 | ⭕                         | ⭕                     | ✅                        | ⭕                   | N/A                                      |
-| Default refresh interval                                                                          | 10s                    | 15s                       | 15s                | 15s                        | 15s                    | 30s                       | 30s                  | 5s                                       |
-| Default metrics interval                                                                          | 60s                    | 60s                       | 60s                | 60s                        | 60s                    | 60s                       | 30s                  | 30s                                      |
+| Capability                                                                                        | [Java](/sdks/java_sdk) | [Node.js](/sdks/node_sdk) | [Go](/sdks/go_sdk) | [Python](/sdks/python_sdk) | [Ruby](/sdks/ruby_sdk) | [.NET](/sdks/dot_net_sdk) | [PHP](/sdks/php_sdk) | [Unleash Proxy Server](unleash-proxy.md) | [Rust](/sdks/rust_sdk) |
+|---------------------------------------------------------------------------------------------------|:----------------------:|:-------------------------:|:------------------:|:--------------------------:|:----------------------:|:-------------------------:|:--------------------:|:----------------------------------------:|:----------------------------------------:|
+| **Category: Initialization**                                                                      |                        |                           |                    |                            |                        |                           |                      |                                         |                                          |
+| Async initialization                                                                              | ✅                     | ✅                        | ✅                 | ✅                         | ✅                     | ✅                        | ✅                   | N/A                                      | ✅                                        |
+| Can block until synchronized                                                                      | ✅                     | ✅                        | ⭕                 | ⭕                         | ⭕                     | ✅                        | ⭕                   | N/A                                      | ⭕                        |
```

Comment:
Oh, can it not, then? So you cant `.await` the init sequence? Huh.

Reply:
This is one of those "depends on what you mean" scenarios. Because of the way Rust, memory and threads work there's no single way to spin up the SDK - the consumer needs to both start it up and make a plan to register a task to do the updates on another thread/async task/whatever you feel like. This also means there's no obvious way to run a function and know that the SDK is synced (because that's your job as a consumer of the SDK). The demos do this by spinning up an async task to poll for updates and waiting half a second in the callbacks for sync to complete. This all makes this feel like a "no" to me but tell me if you think I'm wrong here

Reply:
Oh, right! Now that you mention it, I think I remember seeing that! Yeah, in that case it feels like a "no", indeed. I wonder if we could improve that. I don't have enough insight into how it works, but it might be something to think about, at least 🤔